stl/inc/spanstream

```diff
@@ -55,7 +55,7 @@ public:
     }
 
     basic_spanbuf(const basic_spanbuf&) = delete;
-    basic_spanbuf(basic_spanbuf&& _Right)
+    basic_spanbuf(basic_spanbuf&& _Right) noexcept

```

Comment:
For stream classes with their own buffers, unfortunately, we always copy constructs the `basic_streambuf` base class subobjects, and thus the move constructors can't be `noexcept`. Ditto for `basic_meowbuf` and `basic_(i|o)meowstream` classes.